llvm/include/llvm/CodeGen/SelectionDAGNodes.h

```diff
@@ -932,6 +932,9 @@ END_TWO_BYTE_PACK()
   /// Helper method returns the APInt of a ConstantSDNode operand.
   inline const APInt &getConstantOperandAPInt(unsigned Num) const;
 
+  /// Helper method returns the APInt value of a ConstantSDNode.
+  inline const APInt &getAsAPIntVal() const;
+
```

Comment:
Why not directly put the definition here?

Reply:
The definition of ConstantSDNode doesn't come until later in the file, and this matches the approach used by the neighbouring getters.